mso/resource_mso_schema.go

```diff
@@ -50,18 +74,35 @@ func resourceMSOSchemaCreate(d *schema.ResourceData, m interface{}) error {
 	log.Printf("[DEBUG] Schema: Beginning Creation")
 	msoClient := m.(*client.Client)
 	name := d.Get("name").(string)
-	templateName := d.Get("template_name").(string)
-	tenandId := d.Get("tenant_id").(string)
-
-	schemaApp := models.NewSchema("", name, templateName, tenandId)
+	var schemaApp *models.Schema
```

Comment:
Where do you check if neither template_name nor templates is provided?

Reply:
I'll include it on my next commit